src/Data/RdapEntity.php

```diff
@@ -48,9 +64,7 @@ public function __construct(string $key, $content) {
                     foreach ($vcard as $v) {
                         if (is_array($v)) {
                             foreach ($v as $card) {
-                                if ((is_array($card)) && (count($card)>3)) {
-                                    $this->vcards[] = new RdapVcard($card[0], $card[1], $card[2], $card[3]);
-                                }
+                                $this->vcards[] = new RdapVcard($card[0], $card[1], $card[2], $card[3]);
```

Comment:
reverting the previous fix will generate errors again.
It is not safe to assume $card is an array.